app/javascript/hyacinth_ui_v1/components/digital_objects/rights/ItemRightsForm.js

```diff
@@ -136,10 +51,111 @@ class ItemRightsForm extends React.Component {
     }));
   }
 
-  onSubmitHandler() {
-    // To be implemented.
+  onSubmitHandler = () => {
+    const { data: { uid } } = this.props;
```

Comment:
Lines 55 & 56 can be combined into one, if you'd like.

Reply:
Good suggestion.  Thanks!